Makefile.build

```diff
@@ -88,7 +88,7 @@ include Makefile.dev
 
 .PHONY: world byte coq world.timing.diff coq.timing.diff states
 
-world: coq coqide documentation revision
+world: coq $(if $(filter-out no,$(HASCOQIDE)),coqide,) documentation revision
```

Comment:
It would a bit more verbose, but easier to read, if you use a toplevel if as in the other file